src/app/actions/editTweet.ts

```diff
@@ -0,0 +1,20 @@
+'use server';
+
+import { createClient } from '@/utils/createClient';
+import { redirect } from 'next/navigation';
+
+export const handleEditTweet = async (id: string, formData: FormData) => {
+	const keelClient = createClient();
+	const content = formData.get('content')?.toString() ?? '';
+
+	const response = await keelClient.api.mutations.updateTweet({
+		where: { id },
+		values: { content },
+	});
+
+	if (response.data) {
+		redirect('/auth/home');
+	} else {
```

Comment:
After editing tweet, shouldn't it refresh the page to show the updated tweet instead of redirecting to home?

Reply:
The edit tweet is a separate page. After editing, we need to redirect users to the main page `/auth/home`